redshift_connector/credentials_holder.py

```diff
@@ -135,7 +150,16 @@ def get_expiration(self: "CredentialsHolder") -> datetime.datetime:
         return self.expiration
 
     def is_expired(self: "CredentialsHolder") -> bool:
-        return datetime.datetime.now() > self.expiration.replace(tzinfo=None)
+        utc_offset = time.localtime().tm_gmtoff // 60 // 60
+        now = datetime.datetime.now().astimezone()
+
+        _logger.debug(
+            "Credentials will expire at {} (UTC), local time is {}.\nDifference between local time and UTC time is {} hours.\n".format(
+                self.expiration, now, utc_offset
+            )
+        )
+
+        return (now - datetime.timedelta(hours=utc_offset)) > self.expiration
```

Comment:
From the datetime docs, [here](https://docs.python.org/3/library/datetime.html#datetime.datetime.utcnow) I think we could use `datetime.now(timezone.utc)` instead of manually applying the utc offset.

Reply:
A much better solution! Implemented in 922d7b3